### PR TITLE
maint-5.6 merge 2020-02-20

### PR DESCRIPTION
**Pull request by fischer-ncar**
Merge in latest changes for maint-5.6 branch.
    - Machine upates for izumi, aleph, and stampede.
    - Fix domain filepath for inputdata download check.
    - Add server timeouts for inputdata downloads.
    - Adjust user requested walltime to match expected format.
    - Updates for archive_metadata script.
    - Fix chmod command in safe_copy.
    - Updated JRA forcing is now available with DATM%JRA-1p4-2018, DROF%JRA-1p4-2018 in
       compset longname.  I tested ERS_Ld5.TL319_g17.GIAF_JRA.cheyenne_intel.pop-5day_tavg
       to insure that DATM%JRA, DROF%JRA remain bit for bit.
    - Initialize albdif and albdir with reasonable values if namelist isn't read.
    - Fix mpi_bcast in seq_flux_mct to make sure it only uses valid comm.

This will be the last maint-5.6 merge.  Moving forward, PRs to maint-5.6 will also require a PR to
master.

Test suite:  scripts_regression_tests, ERS_Ld5.TL319_g17.GIAF_JRA.cheyenne_intel.pop-5day_tavg
Test baseline: cime d7d30df7
Test namelist changes: 
Test status: bit for bit

Fixes [CIME Github issue #]

User interface changes?: 

Update gh-pages html (Y/N)?:

Code review:

### CODE REVIEW

File: src/components/data_comps/datm/cime_config/namelist_definition_datm.xml [hunk 1]
```diff
@@ -40,7 +40,8 @@
     CLM1PT		= Run with supplied single point data (force CLM)
     CORE2_NYF		= CORE2 normal year forcing (for forcing POP and CICE)
     CORE2_IAF		= CORE2 intra-annual year forcing (for forcing POP and CICE)
-    CORE_IAF_JRA		= JRA55 intra-annual year forcing (for forcing POP and CICE)
+    CORE_IAF_JRA	= JRA55 intra-annual year forcing (for forcing POP and CICE)
+    CORE_IAF_JRA_1p4_2018 = JRA55 intra-annual year forcing, v1.4, through 2018 (for forcing POP and CICE)
```
Comment (rljacob) [inline src/components/data_comps/datm/cime_config/namelist_definition_datm.xml]: **Is** there something specific regarding POP and CICE in this forcing file?

**Comment by rljacob** [comment]
@jonbob take a look at the changes in /datm/cime_config/namelist_definition_datm.xml

File: src/components/data_comps/datm/cime_config/namelist_definition_datm.xml (hunk 1 above)
Comment (fischer-ncar) [reply src/components/data_comps/datm/cime_config/namelist_definition_datm.xml]: These are the ATM and ROF forcing files used by POP and CICE.

Comment (jonbob) [reply src/components/data_comps/datm/cime_config/namelist_definition_datm.xml]: But I think they are also usable by the MPAS components as well

Comment (rljacob) [reply src/components/data_comps/datm/cime_config/namelist_definition_datm.xml]: DATM, and these new JRA options, can force any ocean or sea-ice model coupled with CIME.    At some point, those comments should be updated but don't have to do that now.

**Comment by billsacks** [comment]
@fischer-ncar thanks for adding me as a reviewer. However, given the reviews by others, I think I'm going to pass on this one unless there are specific things you'd like me to look at.

**Pull request merged**